db-engine/src/main/java/com/splicemachine/db/iapi/sql/compile/RequiredRowOrdering.java

```diff
@@ -44,83 +44,87 @@
+     *                            the right order but there may be duplicates),
+     *            NOTHING_REQUIRED is no operation is required
+     *
+     * @exception StandardException        Thrown on error
+     */
+    int sortRequired(RowOrdering rowOrdering, OptimizableList optimizableList)  throws StandardException;
 
-	/**
-	 * Tell whether sorting is required for this RequiredRowOrdering,
-	 * given a RowOrdering representing a partial join order, and
-	 * a bit map telling what tables are represented in the join order.
-	 * This is useful for reducing the number of cases the optimizer
-	 * has to consider.
-	 *
-	 * @param rowOrdering	The order of rows in the partial join order
-	 * @param tableMap		A bit map of the tables in the partial join order
-	 * @param optimizableList	The current join order being considered by 
-	 *    the optimizer. We need to look into this to determine if the outer
-	 *    optimizables are single row resultset if the order by column is
-	 *    on an inner optimizable and that inner optimizable is not a one
-	 *    row resultset. DERBY-3926
-	 *
-	 * @return	SORT_REQUIRED if sorting is required,
-	 *			ELIMINATE_DUPS if no sorting is required by duplicates
-	 *							must be eliminated (i.e. the rows are in
-	 *							the right order but there may be duplicates),
-	 *			NOTHING_REQUIRED is no operation is required
-	 *
-	 * @exception StandardException		Thrown on error
-	 */
-	int sortRequired(RowOrdering rowOrdering,JBitSet tableMap,OptimizableList optimizableList) throws StandardException;
+    /**
+     * Tell whether sorting is required for this RequiredRowOrdering,
+     * given a RowOrdering representing a partial join order, and
+     * a bit map telling what tables are represented in the join order.
+     * This is useful for reducing the number of cases the optimizer
+     * has to consider.
+     *
+     * @param rowOrdering    The order of rows in the partial join order
+     * @param tableMap        A bit map of the tables in the partial join order
+     * @param optimizableList    The current join order being considered by
+     *    the optimizer. We need to look into this to determine if the outer
+     *    optimizables are single row resultset if the order by column is
+     *    on an inner optimizable and that inner optimizable is not a one
+     *    row resultset. DERBY-3926
+     *
+     * @return    SORT_REQUIRED if sorting is required,
+     *            ELIMINATE_DUPS if no sorting is required by duplicates
+     *                            must be eliminated (i.e. the rows are in
+     *                            the right order but there may be duplicates),
+     *            NOTHING_REQUIRED is no operation is required
+     *
+     * @exception StandardException        Thrown on error
+     */
+    int sortRequired(RowOrdering rowOrdering,JBitSet tableMap,OptimizableList optimizableList) throws StandardException;
```

Comment:
Since we're at spacing, what about spaces after commas?

Reply:
Done